wiki/en/FAQ.md

```diff
@@ -28,7 +28,7 @@ Bear in mind also that all participants should follow [The Golden Rule](/wiki/Cl
 
 ### How do I know if I can join a Server? Are there rules?
 
-In general, if somebody lists a Server on one of the Public Directories provided by Jamulus by default, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms built in. However, some Servers may state their policies in the welcome message you will see in the chat window.
+If somebody lists a Server on one of the Directories built into Jamulus, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. Whilst some Servers may state their policies in the welcome message you will see in the chat window, Jamulus itself does nothing to enforce these. Where more privacy is required by a Server, alternatives to the bult-in Directories are available.
```

Comment:
Which alternatives? I'd link e.g. unregistered servers or something else. It's not good in my opinion to just state, yeah, there are alternatives, but go and find out which they are by yourself. Either don't write this sentence or link the alternatives.

Reply:
OK let's remove it to avoid being caught up in the whole public/private thing.